policies/new_chapter_policy.md

```diff
@@ -0,0 +1,136 @@
+# PyLadies Policy Proposal: New PyLadies Chapter Policy
+
+| Metadata |                                            |
+| -------- | ------------------------------------------ |
+| PPP      | 1                                          |
+| Version  | 1.0                                        |
+| Title    | New PyLadies Chapter Policy                |
+| Authors  | Mariatta <mailto:mariatta@pyladies.com>    |
+| Status   | Draft                                      |
+
+                                
+# TL;DR:
+
+The policy should answer the following questions
+
+1. Why is the policy necessary
+2. What is the purpose of a PyLadies chapter? What does it mean to start a PyLadies chapter? 
+3. What are minimum responsibilities and expectation when someone requested a PyLadies chapter?
+4. What kind of support can new chapter leaders expect from Global PyLadies leadership team?
+5. What questions do we ask in our form?
+6. How do we review new chapter requests?
+7. For accountability, how do we document new chapter creations?
+
```

Comment:
Is this the place to discuss when chapters go stale or new organizers request to take over chapters that were once established but not active? Or is transfer/sunset of chapters discussed elsewhere (given that this is a "New Chapter" policy)?

Reply:
I have another task for "inactive chapter" https://github.com/pyladies/project-admin-logisitics/issues/10 

I wanted to start small with this proposal, and focus on the "new" requests. However, if we think it makes sense to include process surrounding chapter transfer, I'm open to that.